io.catenax.week_based_capacity_group/3.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,424 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
```

Comment:
The copyright should be 2023-2024 according to the latest discussions
```suggestion
# Copyright (c) 2023-2024 BASF SE
```

Reply:
done